packages/mobile/src/exchange/ExchangeReview.tsx

```diff
@@ -87,14 +90,23 @@ class ExchangeReview extends React.Component<Props> {
   componentDidMount() {
     const { makerToken, makerAmount } = this.getConfirmationInput()
     this.props.fetchExchangeRate(makerAmount, makerToken)
+    this.props.fetchTobinTax(makerAmount, makerToken)
```

Comment:
In the time btwn when this is dispatched, and when the value comes in, the old value will still live in the store and be shown in the UI right? Does the UI block until that's in? Or can a user proceed with the incorrect value? 
Ditto concern for `fetchExchangeRate` above. We may need a solution like what we do for fees, where things block while those are refreshed.

Reply:
Good point- created an issue to track this here https://github.com/celo-org/celo-monorepo/issues/2472